src/test/regress/sql/segspace.sql

```diff
@@ -327,3 +327,25 @@ end;
 $func$ language plpgsql;
 
 select workset_cleanup_test();
+
+------------ Ensure that tuplestore is destroyed correctly in a case of error -------------------
+
+create table testdata as
```

Comment:
how about to add ignore block with contents: drop table testdata if exists?

Reply:
No, I only meant only that
```sql
-- start_ignore
drop table if exists testdata;
-- end_ignore
```
and don't place ignore block in out

Reply:
Done

Reply:
No, I only meant only that put
```sql
-- start_ignore
drop table if exists testdata;
-- end_ignore
```
before create table and don't place ignore block in out